message/validation/rsa.go

```diff
@@ -28,26 +24,17 @@ func (mv *messageValidator) verifyRSASignature(messageData []byte, operatorID sp
 			return e
 		}
 
-		operatorPubKey, err := base64.StdEncoding.DecodeString(string(operator.PublicKey))
+		rsaPubKey, err = keys.PublicKeyFromString(string(operator.PublicKey))
```

Comment:
One point of this PR is that now the rest of the code that is not in `keys` doesn't care about which priv/pub key the operator uses. we can now remove traces of the RSA name from this function.